canonical/rebac-admin
# Serve mock API over http

## Done

- Added a script to serve the MSW API over HTTP.
- Added docs for how to use the API.

## QA

- Follow the new steps in the docs to run the API.
- Use the code in the QA instructions of this PR to add an API call: https://github.com/canonical/rebac-admin/pull/24.
- Run `yarn build`.
- In the dashboard check out the rebac-admin branch.
- Link rebac-admin (and React etc.) in the dashboard (instructions in the hacking doc of this project).
- Follow the new steps to set the API url in the ReBACAdmin component.
- Run the dashboard, go to Permissions and check that you can make an API call.
